rides/templates/extend/email.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+
+{% block head %}
+    <head>
+        <title>CSH Rideboard</title>
+        <link rel="stylesheet" 
+              href="https://themeswitcher.csh.rit.edu/api/get" 
```

Comment:
I'm not sure this is going to work with cookies, but we'll see :eyes:

Reply:
Themeswitcher should handle it done.